Dockerfile.simpleserialize

```diff
@@ -0,0 +1,18 @@
+FROM node:lts as build
```

Comment:
Naming it simpleserialize.Dockerfile would trigger proper syntax highlighting in Github?

Reply:
Roger that!!